packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.tsx

```diff
@@ -39,61 +29,43 @@ export type CursorInfo = {
   after?: string;
 };
 
-export type TextEditorSlots = {
-  root?: Omit<ComponentProps<'div'>, 'ref'>;
-  editor?: {
-    className?: string;
-  };
-  content?: {
-    className?: string;
-  };
-};
-
 export type TextEditorProps = {
-  model: EditorModel; // TODO(burdon): Optional (e.g., just provide content if readonly).
-  readonly?: boolean; // TODO(burdon): Move into model.
+  dataTestId?: string; // TODO(burdon): Generalize?
+  doc?: string; // TODO(burdon): Rename text/value/content?
+  className?: string;
   autoFocus?: boolean;
-  scrollPastEnd?: boolean;
-  moveToEndOfLine?: boolean;
-  lineWrapping?: boolean;
   scrollTo?: StateEffect<any>; // TODO(burdon): Restore scroll position: scrollTo EditorView.scrollSnapshot().
+  moveToEndOfLine?: boolean;
   selection?: { anchor: number; head?: number };
-  placeholder?: string;
-  theme?: ThemeStyles;
-  slots?: TextEditorSlots;
   extensions?: Extension[];
   debug?: boolean;
 };
 
 /**
- * Base text editor.
+ * Thin wrapper for text editor.
+ * Handles tabster and focus management.
  */
-// TODO(burdon): Replace with useTextEditor.
```

Comment:
not expecting to replace this anymore? is there something here that necessarily differentiates it from `useTextEditor`?